From 12aba6c4dfafe191a4640e3ab755a1c7e2ddfc44 Mon Sep 17 00:00:00 2001
From: Friedemann Kleint <Friedemann.Kleint@qt.io>
Date: Tue, 15 Oct 2024 08:49:09 +0200
Subject: [PATCH] PySide6/Documentation: Name the .rst doc files according to
 the module name

This makes installing them easier.
Amends b55e221464c18053fa44f18132071ebdaee8f432.

Task-number: PYSIDE-2893
Task-number: PYSIDE-1106
Change-Id: Ic93a1db34ca778ba1df8065c87e51e64e01c5900
Reviewed-by: Shyamnath Premnadh <Shyamnath.Premnadh@qt.io>
(cherry picked from commit 9214d569d1e0765435a0df6a81a3996b93b3d044)
Reviewed-by: Qt Cherry-pick Bot <cherrypick_bot@qt-project.org>
---
 sources/pyside6/PySide6/QtQml/typesystem_qml.xml | 16 ++++++++--------
 .../doc/{qtqml_functions.rst => qtqml.rst}       |  0
 2 files changed, 8 insertions(+), 8 deletions(-)
 rename sources/pyside6/PySide6/doc/{qtqml_functions.rst => qtqml.rst} (100%)

diff --git a/sources/pyside6/PySide6/QtQml/typesystem_qml.xml b/sources/pyside6/PySide6/QtQml/typesystem_qml.xml
index f2e9707c3a..11eb27134e 100644
--- a/sources/pyside6/PySide6/QtQml/typesystem_qml.xml
+++ b/sources/pyside6/PySide6/QtQml/typesystem_qml.xml
@@ -58,42 +58,42 @@
 
     <add-function signature="qmlRegisterType(PyTypeObject@type_obj@,const char*@uri@,int@version_major@,int@version_minor@,const char*@qml_name@)" return-type="int">
         <inject-code class="target" file="../glue/qtqml.cpp" snippet="qmlregistertype"/>
-        <inject-documentation format="target" mode="append" file="../doc/qtqml_functions.rst"
+        <inject-documentation format="target" mode="append" file="../doc/qtqml.rst"
                               snippet="qmlregistertype"/>
         <modify-argument index="2" pyi-type="str"/>
     </add-function>
 
     <add-function signature="qmlRegisterSingletonType(PyTypeObject@type_obj@,const char*@uri@,int@version_major@,int@version_minor@,const char*@qml_name@,PyObject*@callback@)" return-type="int">
       <inject-code class="target" file="../glue/qtqml.cpp" snippet="qmlregistersingletontype_qobject_callback"/>
-      <inject-documentation format="target" mode="append" file="../doc/qtqml_functions.rst"
+      <inject-documentation format="target" mode="append" file="../doc/qtqml.rst"
                             snippet="qmlregistersingletontype_qobject_callback"/>
       <modify-argument index="2" pyi-type="str"/>
     </add-function>
 
     <add-function signature="qmlRegisterSingletonType(PyTypeObject@type_obj@,const char*@uri@,int@version_major@,int@version_minor@,const char*@qml_name@)" return-type="int">
       <inject-code class="target" file="../glue/qtqml.cpp" snippet="qmlregistersingletontype_qobject_nocallback"/>
-      <inject-documentation format="target" mode="append" file="../doc/qtqml_functions.rst"
+      <inject-documentation format="target" mode="append" file="../doc/qtqml.rst"
                             snippet="qmlregistersingletontype_qobject_nocallback"/>
       <modify-argument index="2" pyi-type="str"/>
     </add-function>
 
     <add-function signature="qmlRegisterSingletonType(const char*@uri@,int@version_major@,int@version_minor@,const char*@qml_name@,PyObject*@callback@)" return-type="int">
       <inject-code class="target" file="../glue/qtqml.cpp" snippet="qmlregistersingletontype_qjsvalue"/>
-      <inject-documentation format="target" mode="append" file="../doc/qtqml_functions.rst"
+      <inject-documentation format="target" mode="append" file="../doc/qtqml.rst"
                             snippet="qmlregistersingletontype_qjsvalue"/>
       <modify-argument index="1" pyi-type="str"/>
     </add-function>
 
     <add-function signature="qmlRegisterSingletonInstance(PyTypeObject@type_obj@,const char*@uri@,int@version_major@,int@version_minor@,const char*@qml_name@,PyObject*@callback@)" return-type="int">
       <inject-code class="target" file="../glue/qtqml.cpp" snippet="qmlregistersingletoninstance"/>
-      <inject-documentation format="target" mode="append" file="../doc/qtqml_functions.rst"
+      <inject-documentation format="target" mode="append" file="../doc/qtqml.rst"
                             snippet="qmlregistersingletoninstance"/>
       <modify-argument index="2" pyi-type="str"/>
     </add-function>
 
     <add-function signature="qmlRegisterUncreatableType(PyTypeObject@type_obj@,const char*@uri@,int@version_major@,int@version_minor@,const char*@qml_name@,const char*@message@)" return-type="int">
         <inject-code class="target" file="../glue/qtqml.cpp" snippet="qmlregisteruncreatabletype"/>
-        <inject-documentation format="target" mode="append" file="../doc/qtqml_functions.rst"
+        <inject-documentation format="target" mode="append" file="../doc/qtqml.rst"
                               snippet="qmlregisteruncreatabletype"/>
         <modify-argument index="2" pyi-type="str"/>
     </add-function>
@@ -229,7 +229,7 @@
             </modify-argument>
             <inject-code class="target" file="../glue/qtqml.cpp"
                          snippet="qqmlengine-singletoninstance-qmltypeid"/>
-            <inject-documentation format="target" mode="append" file="../doc/qtqml_functions.rst"
+            <inject-documentation format="target" mode="append" file="../doc/qtqml.rst"
                                   snippet="qqmlengine-singletoninstance-qmltypeid"/>
         </add-function>
         <add-function signature="singletonInstance(QString@uri@,QString@typeName@)"
@@ -241,7 +241,7 @@
             </modify-argument>
             <inject-code class="target" file="../glue/qtqml.cpp"
                          snippet="qqmlengine-singletoninstance-typename"/>
-            <inject-documentation format="target" mode="append" file="../doc/qtqml_functions.rst"
+            <inject-documentation format="target" mode="append" file="../doc/qtqml.rst"
                                   snippet="qqmlengine-singletoninstance-typename"/>
         </add-function>
     </object-type>
diff --git a/sources/pyside6/PySide6/doc/qtqml_functions.rst b/sources/pyside6/PySide6/doc/qtqml.rst
similarity index 100%
rename from sources/pyside6/PySide6/doc/qtqml_functions.rst
rename to sources/pyside6/PySide6/doc/qtqml.rst
